examples/nlp/language_modeling/megatron_retro_fine_tune.py

```diff
@@ -1,144 +0,0 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
we don't need retro model fine tuning?

Reply:
Hi Yi Dong,
Regard this, I didn't mean to remove the current RETRO NeMo implementation and features. Locally I changed them to `*_legacy.py`. In this PR, I didn't put in all the RETRO `*_legacy` files to avoid distraction. I will put them back to not cause confusion.
Besides that, we indeed need some discussion about putting NeMo RETRO and mcore RETRO. I'll start a Slack thread.

Reply:
I have re-added the previous NeMo version files and changed them to `*_legacy`. I also have changed the import of relevant files to not break the code.